posts/continuous_delivery_tools_revisited.md

```diff
@@ -0,0 +1,59 @@
+# Continuous Delivery Tools Revisited
+
+Since we wrote the original [article](https://www.madetech.com/blog/continuous-delivery-tools) just a little over two years ago, we've seen a fairly big shift away from self-hosted tools to feature rich Software as Service (SaaS). We've also seen nearly all Continuous Integration (CI) tools blur their lines with Continuous Delivery (CD) providing an all-in-one solution.
+
+We'll see who the new players are, what are the new features to look for, and then review the advice we gave regarding choosing self-hosted over SaaS.
+
```

Comment:
I think we're missing a sentence here to explain why we're talking about Self-hosted or Saas. What is the angle for the next two sections? are we comparing?